games-action/cylindrix/cylindrix-1.0.recipe

```diff
@@ -1,19 +1,58 @@
-DESCRIPTION="Cylindrix is a 3 degrees of freedom combat game"
-HOMEPAGE="http://www.hardgeus.com/cylindrix/"
-SOURCE_URI="http://www.hardgeus.com/cylindrix/cylindrix-1.0.tar.gz"
+SUMMARY="A combat game"
+DESCRIPTION="Cylindrix is a 3 degrees of freedom combat game" 
+HOMEPAGE="http://www.hardgeus.com/cylindrix/" 
+COPYRIGHT="2001 John R. McCawley III, Anthony Thibault"
+LICENSE="GNU GPL v2"
 REVISION="1"
-CHECKSUM_MD5="914cf70b0a95f4a36cfdfde7edbaa240"
-STATUS_HAIKU="stable"
-DEPEND="media-libs/allegro >= 4.4.1.1"
+SOURCE_URI="http://pkgs.fedoraproject.org/lookaside/pkgs/cylindrix/cylindrix-1.0.tar.gz/914cf70b0a95f4a36cfdfde7edbaa240/cylindrix-1.0.tar.gz"
+CHECKSUM_SHA256="33bfc264d906a1977090e47a38f544551d2939d4c0ea01e1792f63ab4fa06f20"
+
+SOURCE_DIR="cylindrix"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	cylindrix
+	cmd:cylindrix$secondaryArchSuffix
+	"
+
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	lib:liballeg$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:liballeg$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:cmp
+	cmd:aclocal
+	cmd:automake
+	cmd:autoheader
+	cmd:libtoolize
+	cmd:makeinfo
+	"
+
 BUILD()
 {
-	cd cylindrix
-	CFLAGS='-O2' ./configure --prefix=`finddir B_COMMON_DIRECTORY`
-	make
+	libtoolize -fci
+	aclocal -I $sourceDir
+	autoconf
+	automake --add-missing
+	runConfigure ./configure
+	make $jobArgs
 }
 
 INSTALL()
 {
-	cd  cylindrix
 	make install
+	mkdir -p $appsDir/Cylindrix
+	cp cylindrix $appsDir/Cylindrix/Cylindrix
+	cd $sourceDir
+	cp -r 3d_data gamedata pcx_data stats wav_data $appsDir/Cylindrix
+	#addAppDeskbarSymlink $appsDir/Cylindrix/cylindrix Cylindrix
```

Comment:
I haven't test but I think this might work:
```Shell
	runConfigure --omit-dirs binDir ./configure --bindir=$appsDir/Cylindrix
	make $jobArgs
}

INSTALL()
{
	make install
	addAppDeskbarSymlink $appsDir/Cylindrix/cylindrix Cylindrix
}
```